### ISSUE TITLE
Track seek bar gets uselessly dark after a song ends

### ISSUE THREAD

**Issue by kaysond** [airsonic/airsonic#902]
This is on Firefox 65.0.2. Not sure if it affects other browsers as well.

Take a look at https://imgur.com/a/HRdvUPI
The top is when the page first loads. The bottom is after a song finishes and it transitions to the next one.

It looks like the problem is with the code below. On page load, the style is set to `scaleX(0)`. Presumably when the next song loads, it starts this progress bar, but when it finishes, it doesn't reset. The result is two overlaid bars causing the darkness.
```
<span class="mejs__time-loaded" style="transform: scaleX(1);"></span>
```

**Comment by jooola** [comment]
Might be duplicate.

**Comment by kaysond** [comment]
you'd think someone would've reported this already. I did search but couldn't find it

**Comment by jvoisin** [comment]
Any chances you can issue a PR to fix this?

**Comment by kaysond** [comment]
I did try to dig through it a little bit with firefox dev tools but didn't find an immediate answer. My guess is that it's a problem with the mediaelement library. I can keep looking

**Comment by jooola** [comment]
This should have been fixed in 10.3, please test and close the ticket if so.

**Comment by kaysond** [comment]
Doesn't look like it... The behavior has changed slightly though. Now in addition to having this glitch after transitioning to another track, it also happens immediately after playing the first track.

Interestingly, MS Edge does not seem to have the issue on first track play, but I couldn't actually get it to transition to the next track in the play queue without clicking play again... so not sure if it happens between tracks as well
![Untitled](https://user-images.githubusercontent.com/1147328/58121457-60d2b300-7bbc-11e9-84ff-9333f1ce227f.png)

I'll try to debug further

**Comment by muff1nman** [comment]
Seeing this as well.

**Comment by jooola** [comment]
Weird, I just added flac back to the mp3 audio trans-coding rule and I'm seing this as well.

**Comment by tesshucom** [comment]
It seems that the fix has already been applied to the master.
Is it resolved by #1080?
